server/controllers/auth.js

```diff
@@ -0,0 +1,111 @@
+/* eslint-disable no-console */
+/* eslint-disable no-unreachable */
+import jwt from 'jsonwebtoken';
+
+import config from '../config';
+import User from '../models/User';
+
+function invalidResponse(response) {
+  return response.status(400).json({
+    success: false,
+    message: 'Invalid Email/Password.',
+    user: null,
```

Comment:
Do we really need to return `user: null`

Reply:
Ya, that is how we are structuring the responses, had a talk with @avinash360

Reply:
In that case, I think we should change it. The `success` is false and we got the error message. There is no need for `user: null` to be sent. At the frontend, we are not gonna access the `user`, if the `success` is false.